doc/classes/EditorSettings.xml

```diff
@@ -217,6 +220,9 @@
 		<member name="docks/property_editor/subresource_hue_tint" type="float" setter="" getter="">
 			The tint intensity to use for the subresources background in the Inspector dock. The tint is used to distinguish between different subresources in the inspector. Higher values result in a more noticeable background color difference.
 		</member>
+		<member name="docks/scene_tree/ask_before_deleting_related_animation_tracks" type="bool" setter="" getter="">
+			If [code]true[/code], if a deleted node has animation tracks referencing it, the user will be prompted whether they should be deleted or left alone. The confirmation dialog will appear even when using "Delete (No Confirm)" shortcut.
```

Comment:
```suggestion
			If [code]true[/code], when a node is deleted with animation tracks referencing it, a confirmation dialog appears before the tracks are deleted. The dialog will appear even when using "Delete (No Confirm)" shortcut.
```
May also  be worth explaining why not wanting to delete the tracks is desirable sometimes, otherwise keeping this setting off sounds like a net positive (and the setting is deemed to be useless).